example-compose-files/sq-with-postgres/docker-compose.yml

```diff
@@ -19,8 +22,8 @@ services:
     networks:
```

Comment:
I think we should point to specific postgres version especially the one officially supported by us